Projects/Applications/FreeRTOS/FreeRTOS_LoRaWAN/Core/App/Src/receive.c

```diff
@@ -9,4 +11,40 @@ void Process_Rx_Data(const LmHandlerAppData_t *const app_data, const LmHandlerRx
     	APP_LOG(TS_OFF, VLEVEL_M, "%02X ", app_data->Buffer[i]);
     }
     APP_LOG(TS_OFF, VLEVEL_M, "\r\n");
```

Comment:
een lege regel hiertussen zou mooi zijn
```suggestion
    APP_LOG(TS_OFF, VLEVEL_M, "\r\n");
    
```